chore: clarify visibility of insight pages (#273)
open-sauced/docs: docs/features/workspaces.md

![Workspace Access](../../static/img/access-workspace.png)

> By default, all workspaces are public. This means that anyone with the link to your workspace can view it and all of your Insight Pages. If you want to make your workspace private, you can upgrade your account to a paid plan.

Reviewer: Would this be more clarified?

```suggestion
By default, all workspaces are public. This means that anyone with the link to your workspace can view it, including all of your Insight Pages in this workspace. If you want to make your workspace private, you can upgrade your account to a paid plan.
```

> By default, all workspaces are public. This means that anyone with the link to your workspace can view it, including all of your Insight Pages in the workspace. If you want to make your workspace private, you can upgrade your account to a paid plan.

Reviewer: Sorry, I'm missing one thing here 😅

```suggestion
By default, all workspaces are public. This means that anyone with the link to your workspace can view it, including all of your Insight Pages in the workspace. If you want to make your workspace private, you can upgrade your account to a PRO plan.
```

By upgrading to a team workspace, you'll have unlimited contributor and repository insights, as well as the ability to toggle between private and public visibility. 

:::info

> The visibility of your insights pages is determined by the visibility of the workspace. If your workspace is public, your insights will be public. If your workspace is private, your insights will be private.

Reviewer: ```suggestion
The visibility of your Insights Pages is determined by the visibility of the workspace. If your workspace is public, your insights will be public. If your workspace is private, your insights will be private.
```
